test/integration/targets/apt/tasks/main.yml

```diff
@@ -34,5 +34,7 @@
           name: "{{ repodir }}"
           state: absent
 
+  - include: 'apt-error-message.yml'
```

Comment:
Rather than use `include`, use `import_tasks`. I know that is how the other tasks files are imported but they should be updated as well.